src/form/input/__snapshots__/input.test.tsx.snap

```diff
@@ -0,0 +1,17 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Input /> should match snapshot 1`] = `
+<div
+  className="input-container input-container--type-text"
+  data-testid="input"
+>
+  <input
+    autoComplete="off"
+    autoCorrect="off"
+    className="input"
+    name="input"
+    onChange={[Function]}
+    type="text"
+  />
+</div>
```

Comment:
Should we create snapshot instances for all input types? Our `<Input>` components is different than `<input>` element especially for `number` type.